internal/db/rdb/mysql/mysql.go

```diff
@@ -323,7 +326,7 @@ func (m *mySQLClient) SetMetas(ctx context.Context, metas ...MetaVector) error {
 	return tx.Commit()
 }
 
-func deleteMetaWithTx(ctx context.Context, tx *dbr.Tx, uuid string) error {
+func deleteMetaWithTx(ctx context.Context, tx dbr.Tx, uuid string, dbr dbr.DBR) error {
```

Comment:
I think we should use receiver to use `m.DBR` instead.
```suggestion
func (m *mySQLClient) deleteMetaWithTx(ctx context.Context, tx dbr.Tx, uuid string) error {
```